docs/REST API/rest-api.md

```diff
@@ -5676,9 +5676,10 @@ Returns the query report specified by ID.
 
 #### Parameters
 
-| Name | Type    | In   | Description                                |
-| ---- | ------- | ---- | ------------------------------------------ |
-| id   | integer | path | **Required**. The ID of the desired query. |
+| Name      | Type    | In   | Description                                |
+| --------- | ------- | ---- | ------------------------------------------ |
+| id        | integer | path | **Required**. The ID of the desired query. |
+| host_id   | integer | path | If provided, filters to only include results for the specified host. |
```

Comment:
@rachaelshaw we may need an example for clarity, but to confirm it will look like this?

`GET /api/v1/fleet/queries/31/report/439`

Reply:
@mostlikelee whoops, I totally meant for `team_id` to be a query parameter, forgot to change the "In" when I duplicated that line. I just updated the PR; thank you for checking! 
So should look like: `GET /api/v1/fleet/queries/31/report?team_id=439`